server/events.go

```diff
@@ -924,12 +931,6 @@ func (s *Server) remoteServerUpdate(sub *subscription, _ *client, subject, reply
 	}
 	si := ssm.Server
 	node := string(getHash(si.Name))
-	if _, ok := s.nodeToInfo.Load(node); !ok {
```

Comment:
Is this compensated by the addition in processNewServer?

Reply:
Yes.